app/src/features/editor/examples/sway/src20.ts

```diff
@@ -0,0 +1,134 @@
+export const EXAMPLE_SWAY_CONTRACT_SRC20 = `contract;
+
+use ownership;
```

Comment:
@sdankel can we include the entire sway libs as "libs" instead of just the ownership module?

Reply:
`libs` would have to be the name of the package, with all of the modules inside that package. Currently each module in sway-libs is a separate package, so they have to be imported separately. cargo works the same way.